common-lib/src/cmr/common/config.clj

```diff
@@ -216,6 +215,14 @@
   {:default false
    :type Boolean})
 
+(def default-approved-pipeline-documents
+  "The default set of approved pipeline documents"
+  {:grid ["0.0.1"]
+   :dataqualitysummary ["1.0.0"]
+   :orderoption ["1.0.0"]
+   :serviceentry ["1.0.0"]
+   :serviceoption ["1.0.0"]})
```

Comment:
Could you :variable version? or did you remove the reference?

Reply:
Its been removed.